x-pack/plugins/ml/server/routes/results_service.ts

```diff
@@ -352,4 +354,39 @@ export function resultsServiceRoutes({ router, routeGuard }: RouteInitialization
       }
     })
   );
+
+  /**
+   * @apiGroup JobService
+   *
+   * @api {post} /api/ml/results/datafeed_results_chart Get datafeed results chart data
+   * @apiName GetDatafeedResultsChartData
```

Comment:
This needs adding to the `JobService` routes in `apidoc.json`

Reply:
Added in d40f021e1082ac155d793b877b7c0c955a217c53